runtime/westend/src/lib.rs

```diff
@@ -1748,3 +1751,40 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 }
+
+#[cfg(all(test, feature = "try-runtime"))]
+mod remote_tests {
+	use super::*;
+	use frame_try_runtime::runtime_decl_for_TryRuntime::TryRuntime;
+	use remote_externalities::{
+		Builder, Mode, OfflineConfig, OnlineConfig, SnapshotConfig, Transport,
+	};
+
+	#[tokio::test]
+	async fn run_migrations() {
+		sp_tracing::try_init_simple();
+		let transport: Transport = std::option_env!("WSS")
+			.unwrap_or("wss://westend-rpc.polkadot.io:443")
+			.to_string()
+			.into();
+		let maybe_state_snapshot: Option<SnapshotConfig> =
+			std::option_env!("SNAP").map(|s| s.to_string().into());
```

Comment:
Why not parse the env variable at runtime? Why at compile time?

Reply:
no particular preference to be honest, I supposedly had the syntax for this on the top of my head 🤷

Reply:
I think its better to do at runtime, we had CI problems in the past where the CI did not provide the ENV variables.